### ISSUE TITLE
Our AMI does not start on newer instance types (c5d, i3.metal)

### ISSUE THREAD

**Issue by syuu1228** [scylladb/scylla#3465]
Not just i3.metal, but also c5d.*.
It even does not able to start, I got following message just after launched the instance:
```
Launch Failed
The instance configuration for this AWS Marketplace product is not supported. Please see the AWS Marketplace site for more information about supported instance types, regions, and operating systems.
```

**Comment by syuu1228** [comment]
About c5d.*, since we are able to boot on c5.*, we have enough driver to boot up with NVMe, so it must be something else.
Looks like RHEL7 marketplace image does support so tried to find a difference with ours, then I noticed they are using GPT instead of MBR:

```
$ sudo parted /dev/nvme0n1
GNU Parted 3.1
Using /dev/nvme0n1
Welcome to GNU Parted! Type 'help' to view a list of commands.
(parted) p                                                                
Model: NVMe Device (nvme)
Disk /dev/nvme0n1: 10.7GB
Sector size (logical/physical): 512B/512B
Partition Table: gpt
Disk Flags: pmbr_boot

Number  Start   End     Size    File system  Name  Flags
 1      1049kB  2097kB  1049kB                     bios_grub
 2      2097kB  10.7GB  10.7GB  xfs

(parted)  
```

This is Scylla AMI:
```
$ sudo parted /dev/nvme0n1
GNU Parted 3.1
Using /dev/nvme0n1
Welcome to GNU Parted! Type 'help' to view a list of commands.
(parted) p                                                                
Model: Unknown (unknown)
Disk /dev/nvme0n1: 10.7GB
Sector size (logical/physical): 512B/512B
Partition Table: msdos
Disk Flags: 

Number  Start   End     Size    Type     File system  Flags
 1      1049kB  10.7GB  10.7GB  primary  xfs          boot

(parted)  
```
And CentOS Marketplace image does not able to choose c5d.*, i3.metal, it's grayed out.
I think it's likely base image(CentOS official AMI) problem, maybe we have to generate our own base image, again.

**Comment by glommer** [comment]
We got an explanation about that on twitter from an AWS representative.

**Comment by syuu1228** [comment]
@glommer but this is happen even applied initramfs fixes exactly same as what we said: https://twitter.com/_msw_/status/1001311228902322177
And looks like it does not started instance, shows up error message before launching it (maybe there is a validator of the AMI image or something like that).

**Comment by syuu1228** [comment]
@glommer And because of it doesn't looks like started an instance, I don't think scylla_create_devices fixes does not fix current issue: https://twitter.com/_msw_/status/1001315237553848320
I suppose we need fix something to make our image bootable, then we need this.

**Comment by syuu1228** [comment]
Looks like I was mistakenly trying to run older image that is using CentOS Market place AMI as base AMI.
I guess that's why I got "The instance configuration for this AWS Marketplace product..." message.
Using CentOS Market place AMI was mistake, reverted at https://github.com/scylladb/scylla/commit/55d6be92548ed9dcac9e1f14dfb673bd627cbbc1.

I tested again with latest Scylla 2.2 AMI (us-east-1: ami-0ba7c874), I able to run it on c5d.* and i3.metal.
i3.metal takes time to boot up compare to normal VM-based instances, but it just works finely, looks great HW spec.
